client/src/main/java/com/influxdb/client/TaskQuery.java

```diff
@@ -0,0 +1,112 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class TaskQuery {
```

Comment:
Please rename to `TasksQuery`:

```suggestion
public class TasksQuery {
```